### PR TITLE
[orchagent, SRv6]: create seglist support to set sid list type

### PR DESCRIPTION
**Pull request by link19430**
If user doesn't set the sid list type, type will be ENCAPS_RED

What I did
create seglist support to set sid list type

Why I did it
could not set different sidlist type under local testing

How I verified it
set different sidlist type of seg list and confirm it works under local testing

Signed-off-by: link_chiang <link_chiang@edge-core.com>

### CODE REVIEW

**Comment by link19430** [comment]
Hi @prsunny
Could you help to review this PR, thanks

File: orchagent/srv6orch.cpp [hunk 1]
```diff
@@ -365,18 +382,22 @@ void Srv6Orch::doTaskSidTable(const KeyOpFieldsValuesTuple & tuple)
...
     for (auto i : kfvFieldsValues(tuple))
     {
         if (fvField(i) == "path")
         {
           sid_list = fvValue(i);
         }
+        if (fvField(i) == "type")
+        {
+          sidlist_type = fvValue(i);
+        }
```
Comment (kperumalbfn) [inline orchagent/srv6orch.cpp:396]: could you update doc/swss-schema.md with the new attribute 'type' in SRV6_SID_LIST_TABLE

Comment (kperumalbfn) [inline orchagent/srv6orch.cpp:396]: could you update test_srv6.py with this new 'path' attribute in the APP_DB.

Comment (link19430) [reply orchagent/srv6orch.cpp:396]: have added test case for sidlist_type.

Comment (link19430) [reply orchagent/srv6orch.cpp:396]: have updated the swss-schema.md for new attribute 'type' in SRV6_SID_LIST_TABLE

Comment (link19430) [reply orchagent/srv6orch.cpp:396]: @kperumalbfn please help review, thanks.

Comment (link19430) [reply orchagent/srv6orch.cpp:396]: @kperumalbfn please help review, thanks.

**Review by kperumalbfn — APPROVED** [review]

**Pull request merged**